src/EnergyPlus/CrossVentMgr.cc

```diff
@@ -569,11 +577,11 @@ namespace CrossVentMgr {
 
 		// Calculate the opening area for all apertures
 		for ( Ctd = 1; Ctd <= AirflowNetworkSurfaceUCSDCV( 0, ZoneNum ); ++Ctd ) {
-			CompNum = AirflowNetworkLinkageData( Ctd ).CompNum;
-			TypeNum = AirflowNetworkCompData( CompNum ).TypeNum;
-			if ( AirflowNetworkCompData( CompNum ).CompTypeNum == CompTypeNum_DOP ) {
+			EvolveParaUCSDCV_CompNum = AirflowNetworkLinkageData( Ctd ).CompNum;
```

Comment:
And TypeNum isn't even used anywhere - it can go away.  Or it should be set to .CompTypeNum and used in the ifs that follow.